src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");

```

Comment:
The message in the 'doWork' method should indicate the type of Machine that is working. Consider making the message more informative by specifying that it is an Excavator, for example: 'Excavator is starting its work.'